meta-lxatac-bsp/recipes-core/lxatac-repart/lxatac-repart.bb

```diff
@@ -1,3 +1,4 @@
+SUMMARY = "Use systemds update mechanism to only re-partition the disk after image install"
```

Comment:
```suggestion
SUMMARY = "Use systemd's update mechanism to only re-partition the disk after image install"
```

Reply:
Sounds good. I've changed the SUMMARY to match your suggestion.